WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -60,7 +58,7 @@ public async Task<bool> StartCoinJoinAsync(IEnumerable<SmartCoin> coins, Cancell
 			// coordinator with those values. Therefore, conservatively set this
 			// so that a maximum of 5 blame rounds are executed.
```

Comment:
Should this get updated?

```suggestion
			// so that a maximum of 3 blame rounds are executed.
```